src/app/modules/forms/ix-forms/components/ix-explorer/ix-explorer.component.ts

```diff
@@ -63,14 +59,15 @@ import { ErrorParserService } from 'app/services/errors/error-parser.service';
     { ...registeredDirectiveConfig },
   ],
 })
-export class IxExplorerComponent implements OnInit, OnChanges, ControlValueAccessor {
+export class IxExplorerComponent implements ControlValueAccessor {
   readonly label = input<TranslatedString>();
   readonly hint = input<TranslatedString>();
   readonly readonly = input<boolean>(false);
   readonly multiple = input(false);
   readonly tooltip = input<TranslatedString>();
   readonly required = input<boolean>(false);
-  readonly root = input(mntPath);
+  readonly rootNodes = input<Observable<ExplorerNodeData[]>>(/* of([rootDatasetNode]) */);
```

Comment:
I can't think of an advantage of this being an observable. Let's just use array of nodes here.